app/src/main/java/org/sopt/soptseminar/data/githubApi/GithubRepo.kt

```diff
@@ -0,0 +1,10 @@
+package org.sopt.soptseminar.data.githubApi
+
+import com.google.gson.annotations.SerializedName
+
+data class GithubRepo (
+    @SerializedName("name") val name:String,
+    @SerializedName("language") val language:String,
+    @SerializedName("created_at") val date_created:String,
```

Comment:
SerializedName 어노테이션으로 Json형식에 맞춰서 적어주었으니까 우리가 사용하는 객체이름은 carmel case에 맞게 적어주어도 될 것 같아!!
나도 플러그인으로 DataClass일괄적으로 만들었는데, 과제에서 바꾸기 귀찮아서 그냥 냅두긴 했어ㅋㅋㅋㅋ
```suggestion
    @SerializedName("created_at") val dateCreated:String,
```